freebridge-ai/insert_dummy_data.py

```
@@ -0,0 +1,94 @@
+
+            sql_user = """
+            INSERT INTO user (email, name, password, role, email_verified, privacy_agreed, terms_agreed, created_at, updated_at)
+            VALUES (%s, %s, %s, %s, 1, 1, 1, %s, %s)
+            """
+            
+            # 테스트 데이터 생성
+            cursor.execute(sql_user, ('java_expert@test.com', '김철수', 'password', 'FREELANCER', now, now))
+            user_chulsoo_id = cursor.lastrowid
+            
+            cursor.execute(sql_user, ('ai_expert@test.com', '이영희', 'password', 'FREELANCER', now, now))
+            user_younghee_id = cursor.lastrowid
```

Ruff (0.15.2)

[error] Local variable `user_younghee_id` is assigned to but never used

Remove assignment to unused variable `user_younghee_id`

(F841)